test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -263,6 +264,9 @@ test.requestHooks(identifyChallengeMockWithError)('shall clear when session.stat
 
   // Refresh
   await challengeEmailPageObject.refresh();
+  // wait for refresh to complete
+  await t.wait(1000);
+  await t.expect(challengeEmailPageObject.formExists()).eql(true);
```

Comment:
Are both of these needed? I thought that the second assertion would be sufficient since it should wait a certain amount of time for the assertion to become true

Reply:
You might also be able to extend the assertion timeout inline instead of using both